examples/samples/sendersReceiversWithTopics.ts

```diff
@@ -11,7 +11,7 @@ import {
   TopicClient
 } from "../../lib";
 import * as dotenv from "dotenv";
-dotenv.config();
+dotenv.config({ path: "../../.env" });
```

Comment:
All samples run just fine without this change... Let's look at why this is needed in your local setup tomorrow